Libs/GObject/Classes/Object.cs

```diff
@@ -235,6 +240,9 @@ protected static void HandleError(IntPtr error)
 
         protected static IntPtr GetHandle(Object obj)
             => obj.Handle;
+        
+        protected static IntPtr GetHandle(Bytes bytes)
```

Comment:
Yeah, I did not want to make the `Handle` public. This is a little bit of a crux for me. I want to have a nice API meaning in this case no `IntPtr` visible but to achieve this I had to put this very dirty method into `Object`. I'm not satisfied with it either but preferred the nice API here because we tried very hard to hide it beforehand.

I would love to see some other idea which satisfies both needs but I already thought about it quite a lot and see no solution.

I would argue if we make it public for `Bytes` than we allow to leak the native `IntPtr Handle` property into the API. In this case I would remove the `GetHandle(Object obj)` method as well. As the removal would result in cleaner / less code. The API would even be more consistent with it. We probably even should add an `IHandle` interface in this case to make the objects compatible.

Perhaps we should in general decide to go for the cleaner code way and sacrifice a little on the API? If you agree I would introduce the `IHandle` interface and remove both methods.

Reply:
I added the `IHandle` interface.

Reply:
This is a nice solution. Looks good to me.